js/src/routes.js

```diff
@@ -60,6 +62,7 @@ const contractsRoutes = [
 ];
 
 const settingsRoutes = [
+  { path: 'advanced', component: SettingsAdvanced },
```

Comment:
This should go under the Parity tab. However, really should not be introducing new flows a day before release at all. (This should wait until 1.6)